src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -342,12 +342,60 @@ public void TestEvaluateOptionsWithoutAliasMissingRequiredOptions()
             subject.EvaluateOptions().Should().BeFalse();
             this.logTarget.Logs.Should().Contain(new[]
             {
-                "The --resource field is required.",
+                "The --resource field or the --scope field is required.",
                 "The --client field is required.",
                 "The --tenant field is required.",
             });
         }
 
+        /// <summary>
+        /// The test to evaluate options without resource but with scopes.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithOverridedScopes()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = null;
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+            subject.Scopes = new string[] { ".default" };
+
+            subject.EvaluateOptions().Should().BeTrue();
+        }
+
+        /// <summary>
+        /// The test to evaluate options with normal parameters.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithNormalParameters()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = "f0e8d801-3a50-48fd-b2da-6476d6e832a2";
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+
+            subject.EvaluateOptions().Should().BeTrue();
+        }
+
+        /// <summary>
+        /// The test to evaluate options with both resource and scopes.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithResourceAndScopes()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = "f0e8d801-3a50-48fd-b2da-6476d6e832a2";
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+            subject.Scopes = new string[] { ".default" };

```

Comment:
Same here, let's provide a valid scope example.

Reply:
@goagain: I think what @kyle-rader was suggesting here and in the previous comment as well was that the example should contain something more like `f0e8d801-3a50-48fd-b2da-6476d6e832a2/.default`.

Reply:
Or to provide something that isn't `f0e8d801-3a50-48fd-b2da-6476d6e832a2/.default` at all, but rather another valid Azure scope.

Reply:
I just found `scopes` in all existing unit test were `.default`. Do I need to change all of them to `f0e8d801-3a50-48fd-b2da-6476d6e832a2/.default`?